examples/with-cypress-typescript/cypress/plugins/index.ts

```diff
@@ -0,0 +1,31 @@
+import _ from 'lodash'
+import Promise from 'bluebird'
+import axios from 'axios'
```

Comment:
Can we not use any of these 2 libraries to make the example simpler 🙏 

Just native `Promise` and `fetch` may be enough

Reply:
There are also multiple TypeScript errors in this page due to missing types